xwiki-platform-core/xwiki-platform-minimaldependencies/pom.xml

```diff
@@ -200,10 +200,10 @@
      Velocity templates. Most of these templates are located in platform-web and currently we don't declare the
      dependencies of platform-web (they are declared in the distribution) thus we need to bundle this script service
      here. In the future we may want to create a separate module to hold the Velocity templates from platform-web
-     and this module should have a dependency on platform-security-script. -->
+     and this module should have a dependency on platform-security-authorization-script. -->
     <dependency>
       <groupId>org.xwiki.platform</groupId>
-      <artifactId>xwiki-platform-security-script</artifactId>
+      <artifactId>xwiki-platform-security-authorization-script</artifactId>
```

Comment:
Are you sure about this? Are you sure we don't need the authentication script service?

AFAIK `xwiki-platform-security-script` was including all SS before so all your changes need to be reviewed since you dropped the authentication script service from the deps with these changes.

Reply:
Yes I'm sure about it. `security-script` contained `SecurityScriptService` and `SecurityAuthorizationScriptService` but when a package was depending on `security-script` it was because it was depending on `SecurityAuthorizationScriptService`.

Reply:
`AuthenticationScriptService` did not move by the way

Reply:
Indeed, I've checked and `AuthenticationScriptService` was not located inside `security-script`(someone forgot to refactor this apparently in the past, in one way or another ;)).

Reply:
Security module's mess seems to be a tradition ;)